tools/harness-thci/OpenThread_BR.py

```diff
@@ -439,7 +439,7 @@ def ping(self, strDestination, ilength=0, hop_limit=5, timeout=5):
         if hop_limit is None:
             hop_limit = 5
 
-        if self.IsHost or self.IsBackboneRouter:
+        if self.IsHost or self.DeviceCapability == OT12BR_CAPBS or self.DeviceCapability == OT13BR_CAPBS:
```

Comment:
I wonder why we don't use `self.IsBorderRouter`.

Reply:
Good point, I made the change.